aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -24,6 +24,8 @@ DEFINE_DISPATCH(ge_stub);
 DEFINE_DISPATCH(eq_stub);
 DEFINE_DISPATCH(ne_stub);
 DEFINE_DISPATCH(sigmoid_backward_stub);
+DEFINE_DISPATCH(max2_stub);
```

Comment:
can we give this a better name?  Maybe max_elementwise_stub?

Reply:
Done